### DISCUSSION TITLE
zwave js ui 9.0.1 won't connect to aeotec zstick 5+

### DISCUSSION THREAD

**Discussion by shutupfry** [zwave-js/zwave-js#6334] [Request Support / Investigate Issue] [answered]
Everything was working fine. Upgraded the docker image. Suddenly won't connect to the zstick. Downgrade to latest v8 and everything works again.

**Comment by AlCalzone** [comment, marked as answer]
Are you using a VM or USB passthrough of some sort? (Proxmox, Synology VM, ...)
If yes, re-configure it so it handles reconnection of the USB device (some pointers [here](https://github.com/AlCalzone/ioBroker.zwave2/blob/master/docs/en/vm-usb.md)), if not, disable **Soft-Reset** in the Z-Wave settings.

**Reply by shutupfry** [reply]
Yes. It’s being passed through by esxi to a vm.

**Reply by AlCalzone** [reply]
[This thread](https://community.home-assistant.io/t/vmware-esxi-7-0-usb-compatibility/185939) indicates that updating esxi may help.
